@xen-orchestra/backups/_runners/VmsRemote.mjs

```diff
@@ -62,8 +65,19 @@ export const VmsRemote = class RemoteVmsBackupRunner extends Abstract {
         const allSettings = this._job.settings
         const baseSettings = this._baseSettings
 
+        const queue = new Set(vmsUuids)
+        const taskByVmId = {}
+        const nTriesByVmId = {}
+
         const handleVm = vmUuid => {
+          if (nTriesByVmId[vmUuid] === undefined) {
+            nTriesByVmId[vmUuid] = 0
+          }
+          nTriesByVmId[vmUuid]++
```

Comment:
IMHO, it would be much simpler to have a single count which is incremented each time the `asyncMapSettled` is run.

But it's up to you, I can merge it as it is :slightly_smiling_face:

Reply:
But this approach makes it possible to manage the case where a VM has a specific “retry”.

Reply:
True, let's keep it :+1: